packages/cashier/src/pages/p2p-cashier/p2p-cashier.tsx

```diff
@@ -1,15 +1,39 @@
 import React from 'react';
-import PropTypes from 'prop-types';
+import { RouteComponentProps } from 'react-router';
 import { withRouter } from 'react-router-dom';
-import { getLanguage } from '@deriv/translations';
-import { routes, WS } from '@deriv/shared';
-import { Loading } from '@deriv/components';
 import P2P from '@deriv/p2p';
-import { connect } from 'Stores/connect';
+import { Loading } from '@deriv/components';
+import { routes, WS } from '@deriv/shared';
+import { getLanguage } from '@deriv/translations';
 import { get, init, timePromise } from '_common/server_time';
+import { connect } from 'Stores/connect';
+import { RootStore } from 'Types';
+
+type TLocalCurrencyConfig = {
+    currency: string;
+    decimal_places: number;
+};
```

Comment:
@george-usynin-binary We have `RootStore` which everything is typed as `any` there, Instead of creating this local type here I would suggest adding it in the `RootStore` so if someone else is going to use `client.local_currency_config` somewhere else they get the type safety and also we won’t end up with different types for the same thing in different places.

Reply:
@farzin-fs , yeah, that's great notice, will do this 👍